wandering-warriors/main.py

```diff
@@ -1,30 +1,54 @@
 from kivy.app import App
-from kivy.uix.widget import Widget
 from kivy.lang import Builder
+from kivy.uix.screenmanager import ScreenManager, Screen
+from kivy.uix.widget import Widget
 
 
-class ToolManager(Widget):
+class Calculator(Screen):
```

Comment:
What does using `Screen` instead of `Widget` here do? (sorry I don't really know Kivy very well).

Reply:
https://kivy.org/doc/stable/api-kivy.uix.screenmanager.html

Reply:
https://kivy.org/doc/stable/api-kivy.uix.screenmanager.html

Reply:
I don't think ScreenManager is a good choice here.  The app layout isn't dependent on switching between screens, like the checkers practice was.  We'll probably want to leave these as more general Widgets for now.